scripts/validate-urls.py

```diff
@@ -0,0 +1,76 @@
+# This script extract http/https URLs from .md files and validate them.
+
+import argparse
+import os
+import re
+import requests
+
+parser = argparse.ArgumentParser(
+    description='Validate all URLs in the kubeflow.org website'
+)
+
+parser.add_argument(
+    '-d',
+    '--dir',
+    dest='input_dir',
+    nargs='?',
+    default='kubeflow/website/content',
+    help=
+    'Path to the doc conetnt folder. (default: %(default)s)',
```

Comment:
Typo: "conetnt" > "content"

Also, I think "default" should start with a capital letter: 
`(Default: %(default)s)`